src/main/java/net/syscon/elite/service/InmateService.java

```diff
@@ -593,6 +608,13 @@ private void validate(final CategoryApprovalDetail detail) {
         } catch (final EntityNotFoundException ex) {
             throw new HttpClientErrorException(HttpStatus.BAD_REQUEST, "Committee Code not recognised.");
         }
+        if (StringUtils.isNotBlank(detail.getApprovedPlacementAgencyId())) {
+            try {
+                agencyService.getAgency(detail.getApprovedPlacementAgencyId(), ACTIVE_ONLY, "INST");
+            } catch (final EntityNotFoundException ex) {
+                throw new HttpClientErrorException(HttpStatus.BAD_REQUEST, "Review placement agency id not recognised.");
```

Comment:
Maybe could exercise these error conditions in tests?

Reply:
its covered in OffenderAssessmentResourceTest